contracts/adapters/SFCToGovernable.sol

```diff
@@ -2,48 +2,73 @@ pragma solidity ^0.5.0;
 
 import "../model/Governable.sol";
 
+/// @dev SFC is an interface which implements some functions from the SFC required for the Governance contract.
 interface SFC {
-    function getStake(address _from, uint256 _toValidatorID) external view returns (uint256);
+    /// @dev Get the current stake of a delegator for a specific validator.
+    /// @param delegator The address of the delegator.
+    /// @param toValidatorID The ID of the validator to whom the stake is delegated.
+    /// @return The amount of stake delegated by given delegator to the specified validator.
+    function getStake(address delegator, uint256 toValidatorID) external view returns (uint256);
 
-    function getValidator(uint256 _validatorID) external view returns (uint256 status, uint256 deactivatedTime, uint256 deactivatedEpoch,
-        uint256 receivedStake, uint256 createdEpoch, uint256 createdTime, address auth);
+    /// @dev Get information about validator for the given ID.
+    /// @param validatorID The ID of the validator.
+    /// @return status The information about the validator.
+    function getValidator(uint256 validatorID) external view returns (
+        uint256 status,
+        uint256 receivedStake,
+        address auth,
+        uint256 createdEpoch,
+        uint256 createdTime,
+        uint256 deactivatedTime,
+        uint256 deactivatedEpoch
+    );
 
-    function getValidatorID(address _addr) external view returns (uint256);
+    /// @dev Get the current stake of a delegator for a specific validator.
+    /// @param validator The address of the validator.
+    /// @return The ID of the validator.
+    function getValidatorID(address validator) external view returns (uint256);
 
-    function totalActiveStake() external view returns (uint256);
+    /// @dev Get the sum of all active delegated stakes.
+    function getTotalActiveStake() external view returns (uint256);
 }
 
 contract SFCToGovernable is Governable {
```

Comment:
SFCToGovernable is an adapter allowing to use the network SFC contract as Governable (governance votes weights provider).